st2common/st2common/util/param.py

```diff
@@ -216,7 +216,8 @@ def _cast_params_from(params, context, schemas):
     return result
 
 
-def render_live_params(runner_parameters, action_parameters, params, action_context):
+def render_live_params(runner_parameters, action_parameters, params, action_context,
+        additional_contexts=None):
```

Comment:
LGTM.

Let's at least document it needs to be a dict and at what point is added into the final context (e.g. it seems like it's added first and stuff from it could potentially be overwritten from other contexts).

Reply:
And also at least a basic test case :)